src/i_musicpack.c

```diff
@@ -1337,11 +1379,13 @@ static void RestartCurrentTrack(void)
     current_track_pos = file_metadata.start_time;
     SDL_UnlockAudio();
 }
+#endif // !USE_SDL_MIXER_LOOPING
 
 // Poll music position; if we have passed the loop point end position
 // then we need to go back.
 static void I_MP_PollMusic(void)
```

Comment:
I think there could be two versions of this function. One that does things when `!USE_SDL_MIXER_LOOPING` and an empty one otherwise. This should reduce the number of `if`s.

Reply:
Isn't `I_MP_PollMusic` already set up in such a way, a no-op when `!USE_SDL_MIXER_LOOPING`?

Reply:
You currently have one `I_MP_PollMusic` whose contents change depending on the preprocessor define. The alternative is having two different `I_MP_PollMusic` functions, one doing the polling and one doing nothing. The define would then pick which one is used. This is a style thing and could be argued either way so it's not a requirement.